src/current/cockroachcloud/network-authorization.md

```diff
@@ -1,45 +1,52 @@
 ---
-title: Network Authorization
+title: Network Authorization for CockroachDB Cloud Clusters
 summary: Learn about the network authorization features for {{ site.data.products.db }} clusters.
 toc: true
 docs_area: manage
 ---
 
-{{ site.data.products.dedicated }} requires  explicit authorization of the networks that can access your cluster. This deny-by-default policy protects the cluster from network attacks including denial-of-service and password brute-forcing.
+{{ site.data.products.dedicated }} allows explicit configurations of the networks that can access your cluster.
 
-You can do this by:
+To prevent denial-of-service attacks, brute force password attacks, and other forms of malicious activity, Cockroach Labs recommends restricting your network to allow access only from specific IP address ranges controlled by your organization, according to the [Principle of Least Privilege (PoLP)](https://en.wikipedia.org/wiki/Principle_of_least_privilege). These might include specific networks for your application servers, hardened administrator access points, and backup-restore pipelines for disaster recovery.
```

Comment:
```suggestion
To prevent denial-of-service attacks, brute force password attacks, and other forms of malicious activity, Cockroach Labs recommends restricting your network to allow access only from specific IP address ranges controlled by your organization. These might include specific networks for your application deployments, hardened administrator access points, or disaster recovery pipelines.
```